### PR TITLE
Added a question in SQL Quiz

### PR DESCRIPTION
**Pull request by Suhas2002**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL, which of the following is not a commonly used SQL constraints",  
+    Answer: "UNIQUE",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:4]: But this is a constraint. 

```suggestion
    Answer: "UPDATE",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    Distractor1: "CHECK",
+    Distractor2: "DEFAULT",
+    Distractor3: "UPDATE",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:7]: this is supposed to be the correct answer though. remember that distractors are for wrong answers only

```suggestion
    Distractor3: "UNIQUE",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+    Explanation: "In SQL, The UPDATE statement is used to modify the existing records in a table.  Hence, it is not a constraint",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:8]:
```suggestion
    Explanation: "In SQL, the UPDATE statement is used to modify the existing records in a table and therefore not a constraint.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Suhas2002** [comment]
Hey @jdwilkin4, I have made the changes you asked for

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**